redfish-core/src/error_messages.cpp

```diff
@@ -228,8 +228,11 @@ nlohmann::json internalError(void)
                        "consider resetting the service."}};
 }
 
-void internalError(crow::Response& res)
+void internalError(crow::Response& res, const bmcweb::source_location location)
 {
+    BMCWEB_LOG_CRITICAL << "Internal Error " << location.file_name() << "("
```

Comment:
Does BMCWEB_LOG_CRITICAL gets an entry into the journal traces by default ?

Reply:
Yeah, the critical level is higher then the error level so we will get both.